tests/tasks/setup_ipa.yml

```diff
@@ -12,6 +12,7 @@
 - name: Clone ansible-freeipa repo
   git:
     repo: https://github.com/freeipa/ansible-freeipa
+    # wokeignore:rule=master
     version: master
```

Comment:
Here you can omit `version: master` - by default it will grab whatever branch is configured to be the default branch

Reply:
ansible-lint reports this error if no `version: master` is specified.
```
Error: git-latest Git checkouts must contain explicit version
You can skip specific rules or tags by adding them to your configuration file:
# .ansible-lint
warn_list:  # or 'skip_list' to silence them completely
  - git-latest  # Git checkouts must contain explicit version

Finished with 1 failure(s), 0 warning(s) on 82 files.
ERROR: InvocationError for command /home/runner/work/certificate/certificate/.tox/ansible-lint/bin/ansible-lint -v --exclude=tests/roles -c .ansible-lint (exited with code 2)
```
Which do we prefer between having `version: master` preceded by `# wokeignore:rule=master` or adding `- '403'` to the skip-list in `.ansible-lint`?

Reply:
This commit https://github.com/linux-system-roles/certificate/pull/142/commits/604afea5eb3c8dc204709816745c95c31ea80da2 takes the latter.

Reply:
I would prefer noqa latest - https://ansible-lint.readthedocs.io/rules/latest/
```
  # noqa latest
  git:
    repo: https://github.com/freeipa/ansible-freeipa
    # wokeignore:rule=master
    version: master
```
I much, much prefer putting such suppressions inline in the code rather than the config file
* the suppression is next to the problematic code
* easier to deal with conversion to collection - don't have to merge a bunch of .ansible-lint files together
* doesn't affect all files

(/me wishes the error message said this rather than "just add it to the .ansible-lint file" :-(

Reply:
Hmmm., this is the diff I made.
```
@@ -10,6 +10,7 @@
   when: __is_beaker_env
 
 - name: Clone ansible-freeipa repo
+  # noqa latest
   git:
     repo: https://github.com/freeipa/ansible-freeipa
     dest: /tmp/freeipa-repo
```
And I ran `tox -e ansible-lint`, which reported:
```
git-latest: Git checkouts must contain explicit version
tests/tasks/setup_ipa.yml:12 Task/Handler: Clone ansible-freeipa reported

You can skip specific rules or tags by adding them to your configuration file:
# .ansible-lint
warn_list:  # or 'skip_list' to silence them completely
  - git-latest  # Git checkouts must contain explicit version

Finished with 1 failure(s), 0 warning(s) on 86 files.
ERROR: InvocationError for command /home/nhosoi/linux-system-roles/certificate/.tox/ansible-lint/bin/ansible-lint -v --exclude=tests/roles --exclude=.github --exclude=.yamllint.yml -c .ansible-lint (exited with code 2)
```
I wonder `# noqa` works in the yaml file?